### PR TITLE
 Scripts/Spells: Portal to Orgrimmar & Portal to Stormwind 

### PR DESCRIPTION
**Pull request by Sorikoff**
**Changes proposed:**
Migrate Portal to Orgrimmar and Portal to Stormwind (DK chain last quests) to scripts system.

**Target branch(es):** 3.3.5

**Tests performed:** Works.

### CODE REVIEW

**Comment by Sorikoff** [comment]
Yes, yes, reverted revert :smile:

**Comment by jackpoz** [comment]
Could you fix the merge conflicts ? That usually means merging 3.3.5 branch into this 3.3.5-teleport branch, solving manually the merge conflicts by choosing which lines to keep, commit the changes and pushing the branch.

If the merge goes wrong, please don't close/reopen the PR, it can be a good chance to learn something (and we can help too!)

**Comment by Sorikoff** [comment]
Weird, I can't see any conflicts locally.

**Comment by jackpoz** [comment]
can you merge latest 3.3.5 into your branch ? or rebase it, whatever you prefer and feel confident doing

**Comment by Sorikoff** [comment]
Must be fixed now. Thanks for your help!

File: src/server/scripts/EasternKingdoms/ScarletEnclave/chapter5.cpp [hunk 1]
```diff
@@ -1664,8 +1666,37 @@ class npc_the_lich_king_tirion_dawn : public CreatureScript
...
+    PrepareSpellScript(spell_teleport_leaders_blessing);
+
+    bool Load() override
+    {
+        return GetCaster()->GetTypeId() == TYPEID_PLAYER;
+    }
+
+    void HandleScriptEffect(SpellEffIndex /* effIndex */)
+    {
+        Player* target = GetHitPlayer();
```
Comment (jackpoz) [inline src/server/scripts/EasternKingdoms/ScarletEnclave/chapter5.cpp]: The old code checks if the target is a player while the new code checks if the caster is a player. Maybe it's better to add a NULL check here too and return if target is null.

Comment (Sorikoff) [reply src/server/scripts/EasternKingdoms/ScarletEnclave/chapter5.cpp]: I removed caster check since script anyway uses target only.

**Comment by jackpoz** [comment]
Thanks for the PR :)

**Pull request merged**